spacy/tests/regression/test_issue4001-4500.py

```diff
@@ -288,35 +288,33 @@ def set_annotations(self, docs, scores):
     dummy_pipe(doc)
 
 
-@pytest.mark.skip(reason="removed Beam stuff during the Example/GoldParse refactor")
 def test_issue4313():
     """ This should not crash or exit with some strange error code """
     beam_width = 16
     beam_density = 0.0001
     nlp = English()
-    config = {}
-    ner = nlp.create_pipe("ner", config=config)
+    config = {
+        "beam_width": beam_width,
+        "beam_density": beam_density,
+    }
+    ner = nlp.add_pipe("beam_ner", config=config)
     ner.add_label("SOME_LABEL")
-    ner.initialize(lambda: [])
+    nlp.initialize()
     # add a new label to the doc
     doc = nlp("What do you think about Apple ?")
     assert len(ner.labels) == 1
     assert "SOME_LABEL" in ner.labels
+    ner.add_label("MY_ORG")   # TODO: not sure if we want this to be necessary...
```

Comment:
I think this shouldn't be necessary: when we add a previously trained NER to a pipeline with a matcher, you'd need to add the matcher's labels to the NER. Ideally, this test should work without this line.

Reply:
On the other hand the resize-while-running thing is pretty quirky. We have a separate discussion about that, so I'll leave this alone for here.

Reply:
Ok! I've created a quick ticket as a reminder: https://github.com/explosion/spaCy/issues/6681